catboost/python-package/catboost/core.py

```diff
@@ -191,7 +191,7 @@ def _check_data_empty(self, data):
                     data_shape = tuple(data_shape + tuple([len(data[0])]))
                 else:
                     data_shape = tuple(data_shape + tuple([1]))
-            if not len(data_shape) == 2 or not sum(data_shape) > 2:
+            if not len(data_shape) == 2 or not min(data_shape) > 2:
```

Comment:
you mean 0?

Reply:
Yes, I meant 0, as in the issue. Corrected it.